lib/omniauth/strategies/linkedin.rb

```diff
@@ -8,7 +8,8 @@ class LinkedIn < OmniAuth::Strategies::OAuth2
       option :client_options, {
         :site => 'https://api.linkedin.com',
         :authorize_url => 'https://www.linkedin.com/oauth/v2/authorization?response_type=code',
-        :token_url => 'https://www.linkedin.com/oauth/v2/accessToken'
+        :token_url => 'https://www.linkedin.com/oauth/v2/accessToken',
+        :auth_scheme => 'request_body'
```

Comment:
This doesn't change anything for our current main since we're still using oauth 1.X.X and that is a default value, but this allows us to upgrade to 2.X.X in follow on PR (default value is different in 2.X.X). Plus we don't use ui fork of this yet; gonna open PR in main repo to see if we can get rid of ui fork